src/components/UploadPhoto/UploadPhoto.tsx

```diff
@@ -25,17 +26,13 @@ export const UploadPhoto: React.FC<UploadPhotoProps> = ({setUploadedFileData}) =
             reader.readAsDataURL(file);
         }
     };
+
+    const uploadImageButton = { text: 'UPLOAD PHOTO', to: '' };
```

Comment:
Do action buttons need a "to" field? Should we have different types for NavButtons and ActionButtons?  It feels weird to have 1 type but 2 classes. Or maybe we can we make 1 type and 1 class (with an optional "to", "input" and so forth)? I wish we were in Java and could have a non-instanciable interface or abstract class for Button with different classes of buttons inheriting from that. I'm not sure if that's possible in react though.

Reply:
No, they don't need a to field. I was thinking about the Button type more as an interface. I think maybe we should rename the button type to navbutton and not use it in the action button.